EasyGYM/UI/Controllers/Core/LibraryViewController.swift

```diff
@@ -14,15 +14,15 @@ import FirebaseFirestore
 
 class LibraryViewController: UIViewController {
     
+    var viewModel = LibraryViewModel()
+    
```

Comment:
import FirebaseAuth
import Firebase
import FirebaseStorage
import FirebaseFirestore

in Line 10-13 can now be deleted since the fetching is now done in WorkoutService (: